wsf_scraping/pipelines.py

```diff
@@ -69,9 +69,17 @@ def check_keywords(self, item, spider_name, base_pdf_path):
             )
 
             if self.settings['PARSING_METHOD'] == 'pdftotext':
-                pdf_file = parse_pdf_document_pdftxt(f)
+                pdf_file, err = parse_pdf_document_pdftxt(f)
             else:
-                pdf_file = parse_pdf_document(f)
+                pdf_file, err = parse_pdf_document(f)
+
+            if not pdf_file:
+                self.logger.warning(
+                    "The pdf [%s] could not be converted: %s",
+                    item['pdf'],
+                    err,
```

Comment:
Following my previous comment, I am a bit unsure whether this is the right place to throw the warning. Why not return empty pdf_file if there was an error in parsing and throw a warning inside the parsing?